brainstormforce/ultimate-addons-for-gutenberg
# Fix: Forms - Hide background color when layout set to underlined.

### Description

- When field layout is underlined, no need of BG color in Style > Input
- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/c81b814d-1692-4a75-a59e-41bf4b62256c?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop forms set underline layout.
- Check style > input now bg not visible when it set to underline.

### Checklist:
